File: octo/oswatch/src/main/java/com/sankuai/octo/oswatch/model/ConsumerResult.java
package com.sankuai.octo.oswatch.model;

/**
 * Created by chenxi on 7/27/15.
 */
public class ConsumerResult {
    String consumerAppKey;
    double qpsAvg;

    public String getConsumerAppKey() {
        return consumerAppKey;
    }

    public void setConsumerAppKey(String consumerAppKey) {
        this.consumerAppKey = consumerAppKey;
    }

    public double getQpsAvg() {
        return qpsAvg;
    }

    public void setQpsAvg(int qpsAvg) {
        this.qpsAvg = qpsAvg;
    }
}

File: octo/msgp/msgp-server/src/main/scala/com/sankuai/octo/msgp/model/ConsumerResult.scala
package com.sankuai.octo.msgp.model

import play.api.libs.json.Json

/**
 * Created by dreamblossom on 15/9/30.
 */
case class ConsumerResult(consumerAppKey: String, qpsAvg: Double)

object ConsumerResult {
  implicit val consumerResultReader = Json.reads[ConsumerResult]
  implicit val consumerResultWriter = Json.writes[ConsumerResult]
}

